github/repos.go

```diff
@@ -75,6 +75,7 @@ type Repository struct {
 	AssigneesURL     *string `json:"assignees_url,omitempty"`
 	BlobsURL         *string `json:"blobs_url,omitempty"`
 	BranchesURL      *string `json:"branches_url,omitempty"`
+	CodeOfConduct    *string `json:"code_of_conduct,omitempty"`
```

Comment:
yeah. pretty sure my hand is faster than my brain at that moment. Thanks for catching that typo!!! updated.